### PR TITLE
Rename Social Vulnerability to Social Vulnerability Score

### PR DESCRIPTION
**Pull request by ywkim312**
- [x] Need to add how to deprecate
- [x] Add deprecate to requirements
- [x] Add deprecate to setup
- [x] Need to update documentation
- [x] Need to figure out if we change the input data type
- e.g, social_vulnerability_demographic_factors or socialVulnerabilityFeatureAverages. If so, the dataset type should also be changed

### CODE REVIEW

**Review by longshuicy — APPROVED** [review]
Besides the missing copyright year everything looks good.
Both old and new analysis runs and results are identical.
Approving but please add the missing copyright back. Thanks!

File: pyincore/analyses/socialvulnerability/socialvulnerability.py [hunk 1]
```diff
@@ -1,163 +1,19 @@
-# Copyright (c) 2021 University of Illinois and others. All rights reserved.
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply pyincore/analyses/socialvulnerability/socialvulnerability.py:1]: Should be there now

File: pyincore/analyses/socialvulnerabilityscore/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 University of Illinois and others. All rights reserved.
```
Comment (navarroc) [inline pyincore/analyses/socialvulnerabilityscore/__init__.py:1]: This should match the original copyright year, 2021

Comment (ywkim312) [reply pyincore/analyses/socialvulnerabilityscore/__init__.py:1]: I have already updated. Maybe you're using the old one? Can you pull one more time?

Comment (ywkim312) [reply pyincore/analyses/socialvulnerabilityscore/__init__.py:1]: oh my bad, changed it.

**Review by navarroc — APPROVED** [review]

**Pull request merged**